app/models/user.rb

```diff
@@ -93,7 +93,7 @@ def is_unclaimed?
   end
 
   def is_email_pending?
-    !email_addresses.any?(&:verified)
+    !email_addresses.any?(&:verified) && !email_addresses.empty?
   end
```

Comment:
Can we rename this method to make it more clear what it does?  "pending" is a little vague to me.  In fact I'm having a hard time figuring out exactly what it is saying.  Seems like it is returning true if and only if there is at least one email address and none of them are verified.  Maybe just saying `has_emails_pending_verification?` is enough to clear it up?  Hm.. that doesn't account for the added logic.  `has_emails_and_all_unverified?` ?  Better ideas?